epinorm/norm.py

```diff
@@ -122,6 +129,86 @@ def get_feature_ids(self):
             .dropna()
             .unique()
         )
+    
+    def _admin_levels_table(self):
+        """
+        This method retrives the information in the "countries.csv" and "administrative_units.tsv".
+        It returns a dictionnary that maps a country name (from countries.csv) to a list of its administrative levels.
+        administrative levels are encoded as dictionaries with properties:
+        {
+            name,
+            admin_level,
+            osmId
+        }
+
+        it also tranliterates information from the administrative_units.tsv file.
+        """
```

Comment:
A few typos. Also, we (or at least I) try to adhere to the 80 chars line length limit.

```suggestion
        """
        Retrieves the information in "countries.csv" and
        "administrative_units.tsv", and returns a dictionary that maps a
        country name (from "countries.csv") to a list of its administrative
        levels.

        Administrative levels are encoded as dictionaries with properties:
        {
            name,
            admin_level,
            osmId
        }

        Also transliterates information from the administrative_units.tsv file.
        """
```